src/components/ProfileCard/ProfileCard.styles.ts

```diff
@@ -0,0 +1,224 @@
+import { Card, Avatar, Typography, Box } from '@mui/material';
+import { styled } from '@mui/material/styles';
+
+import { PROFILE_CARD_SPACING, PROFILE_CARD_TYPOGRAPHY } from '../../style-library/theme/theme';
+import { VARIANT_CONFIGS } from './ProfileCard.constants';
+
+import type {
+  StyledProfileCardProps,
+  ProfileCardSize,
+  ProfileCardVariant,
+} from './ProfileCard.types';
+
+export const StyledCard = styled(Card, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $size }) => {
+  const spacingConfig = PROFILE_CARD_SPACING[$size];
+
+  return {
+    padding: theme.spacing(spacingConfig.paddingUnits),
+    borderRadius: theme.spacing(2),
+    boxShadow: theme.shadows[2],
+    transition: theme.transitions.create(['box-shadow', 'transform'], {
+      duration: theme.transitions.duration.short,
+    }),
+    '&:hover': {
+      boxShadow: theme.shadows[4],
+      transform: 'translateY(-2px)',
+    },
+    [theme.breakpoints.down('sm')]: {
+      padding: theme.spacing(2),
+    },
+  };
+});
+
+export const StyledProfileContainer = styled(Box, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<StyledProfileCardProps>(({ theme, $variant }) => {
+  const variantConfig = VARIANT_CONFIGS[$variant];
+
+  return {
+    display: 'flex',
+    flexDirection: variantConfig.direction,
+    alignItems: 'center',
+    gap: theme.spacing(2),
+    width: '100%',
+    [theme.breakpoints.down('sm')]: {
+      flexDirection: 'column',
+      gap: theme.spacing(1.5),
+    },
+  };
+});
+
+export const StyledAvatar = styled(Avatar, {
+  shouldForwardProp: (prop) => prop.toString().indexOf('$') !== 0,
+})<{ $size: ProfileCardSize }>(({ theme, $size }) => {
+  const spacingConfig = PROFILE_CARD_SPACING[$size];
+
+  return {
+    width: theme.spacing(spacingConfig.avatarSpacing),
+    height: theme.spacing(spacingConfig.avatarSpacing),
+    backgroundColor: theme.palette.primary.main,
+    fontSize: `calc(${theme.spacing(spacingConfig.avatarSpacing)} * 0.4)`,
+    fontWeight: theme.typography.fontWeightBold || 600,
+    border: `${spacingConfig.borderWidth}px solid ${theme.palette.background.paper}`,
```

Comment:
Border is constructed using a hardcoded `px` unit (`${spacingConfig.borderWidth}px`). The guideline forbids hardcoded px/hex – prefer theme spacing and theme palette tokens.

**Sugerencia:**
```typescript
Use `border: `${theme.spacing(spacingConfig.borderWidth)} solid ${theme.palette.divider}`` (or better, use `theme.palette.divider` for the border color and `theme.spacing(...)` for thickness). Avoid raw `px` strings.
```